PlayerMatcher/packages.config

```diff
@@ -1,27 +1,30 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Antlr" version="3.4.1.9004" targetFramework="net461" />
```

Comment:
it looks like you may have accidentally, or purposefully, updated all of the nuget packages. I bet this is why your view doesn't render properly anymore. try running ```$git checkout  origin/master PlayerMatcher/packages.config``` then run in the package manager ```PM> Update-Package -reinstall``` and rebuilding the solution and see if that fixes the views

Reply:
Yeah looks like that's the root cause. A few iterations ago, the code wouldn't even run without updating the nuget packages, so I'd started just doing that when I did a new branch.